resources/assets/lib/user-list.tsx

```diff
@@ -40,6 +40,14 @@ interface State {
   viewMode: ViewMode;
 }
 
+function rankSortDescending(x: User, y: User) {
+  if (x.current_mode_rank && y.current_mode_rank) {
```

Comment:
Should not really affect anything but `!= null` check avoids javascript casting fun.